Doc/library/re.rst

```diff
@@ -637,6 +637,11 @@ form.
       programs that use only a few regular expressions at a time needn't worry
       about compiling regular expressions.
 
+.. class:: RegexFlag
+
+   An :class:`~enum.Enum` class containing the regex options listed below.
```

Comment:
The tilde creates a link with only the last element visible, i.e. `Enum` here, but since it should be IntFlag, I also removed the tilde as IntFlag is less known (though reader can still hover over shortened name to see where it links of course).